### PR TITLE
Bump resizer images

### PR DESCRIPTION
**Pull request by gnufied**

### CODE REVIEW

File: k8s.gcr.io/images/k8s-staging-sig-storage/images.yaml [hunk 1]
```diff
@@ -116,7 +117,7 @@
     "sha256:92257881c1d6493cf18299a24af42330f891166560047902b8d431fb66b01af5": [ "v1.9.0" ]
 - name: iscsiplugin
   dmap:
-    "sha256:31117074c183242b223340f2fc0f886243e97c6cb1a83d98cfb00a69d1746697": [ "v0.1.0" ]
+    "sha256:31117074c183242b223340f2fc0f886243e97c6cb1a83d98cfb00a69d1746697": [ "0.1-canary;v0.1.0" ]
```
Comment (humblec) [inline k8s.gcr.io/images/k8s-staging-sig-storage/images.yaml:120]: we have to ignore this change and also for nfs and smb plugins

Comment (gnufied) [reply k8s.gcr.io/images/k8s-staging-sig-storage/images.yaml]: done.PTAL

File: k8s.gcr.io/images/k8s-staging-sig-storage/images.yaml [hunk 2]
```diff
@@ -151,15 +152,10 @@
     "sha256:63d5e04551ec8b5aae83b6f35938ca5ddc50a88d85492d9731810c31591fa4c9": [ "v4.0.2" ]
 - name: nfsplugin
   dmap:
-    "sha256:ec71d1797d1af17563bc5d19dfcf0229ca0f631d4a245f64ac358f7499946ee4": [ "v3.0.0" ]
```
Comment (humblec) [inline k8s.gcr.io/images/k8s-staging-sig-storage/images.yaml:154]: we need to ignore these changes too @gnufied

File: k8s.gcr.io/images/k8s-staging-sig-storage/images.yaml [hunk 3]
```diff
@@ -151,15 +152,10 @@
...
-    "sha256:aa38be2f98a9d9c7093b450915bbb2a2b4f53a74f1b6de76cef3aa5c80df45f5": [ "v3.1.0" ]
     "sha256:fded5b914c2531c2471fd0088b75fd33b0756a6c42df4c069007857419e54fa9": [ "v4.0.0" ]
     "sha256:a34c15c963a86da9720090f44c904f49d52b0454784f27017bb863ccbf0537a2": [ "v4.1.0" ]
 - name: smbplugin
   dmap:
-    "sha256:26ca83cac8ea40e2c0c498f681f18df6c0c5e8fa50c3550e20c5bf360dc80120": [ "v1.4.0" ]
-    "sha256:f96d5afd073e937b77638fcea40e664f0d3fce07454cb5a38bfe9397b2f3914c": [ "v1.5.0" ]
-    "sha256:6d8df4a0ed575da16e03ca5ea25b9d060297a747922c4b53a87ad368609f08a1": [ "v1.6.0" ]
```
Comment (humblec) [inline k8s.gcr.io/images/k8s-staging-sig-storage/images.yaml:162]: we need to ignore these changes too @gnufied

**Comment by humblec** [comment]
Cc @xing-yang @msau42

**Pull request merged**